unity-client/Assets/Scripts/MainScripts/DCL/Helpers/Utils/Utils.cs

```diff
@@ -653,5 +653,19 @@ public static bool CompareFloats( float a, float b, float precision = 0.1f )
             key = tuple.Key;
             value = tuple.Value;
         }
+
+        /// <summary>
+        /// This get the renderer bounds with a check to ensure the renderer is at a safe position.
+        /// If the renderer is too far away from 0,0,0, wasm target ensures a crash.
+        /// </summary>
+        /// <param name="renderer"></param>
+        /// <returns>The bounds value if the value is correct, or a mocked bounds object with clamped values if its too far away.</returns>
+        public static Bounds GetSafeBounds( this Renderer renderer )
+        {
+            if ( Vector3.Distance(renderer.transform.position, Vector3.zero) > 1000000 )
+                return new Bounds( Vector3.one * 1000000, Vector3.one * 0.1f );
+
+            return renderer.bounds;
+        }
```

Comment:
question:
Why are we using `Vector3.Distance` here instead of `Abs(renderer.transform.position)`?
suggestion: 
Can we use a local constant for the limit to avoid the same number to be repeated? `const float limit = 1000000;`

Reply:
mmm.. after giving it another though, if we are trying to avoid an overflow, maybe using `sqrMagnitude` it's not the best approach 🤔 😅

Reply:
It's a tiny overflow, not an actual float overflow, I think your suggestion is correct. Ty!.

Reply:
Feedback applied.